src/precompile.jl

```diff
@@ -0,0 +1,88 @@
+Base.precompile(Tuple{typeof(*),SparseArrays.SparseMatrixCSC{Float64, Int64},Matrix{Float64}})   # time: 0.005822867
+Base.precompile(Tuple{typeof(mul!),SubArray{Float64, 1, _B, Tuple{UnitRange{Int64}}, true} where _B<:Union{Base.ReinterpretArray{T, N, S, A, IsReshaped} where {T, N, A<:Union{SubArray{T, N, A, I, true} where {T, N, A<:DenseArray, I<:Union{Tuple{Vararg{Real, N} where N}, Tuple{AbstractUnitRange, Vararg{Any, N} where N}}}, DenseArray}, IsReshaped, S}, SparseArrays.ReshapedArray{T, N, A, MI} where {T, N, A<:Union{Base.ReinterpretArray{T, N, S, A, IsReshaped} where {T, N, A<:Union{SubArray{T, N, A, I, true} where {T, N, A<:DenseArray, I<:Union{Tuple{Vararg{Real, N} where N}, Tuple{AbstractUnitRange, Vararg{Any, N} where N}}}, DenseArray}, IsReshaped, S}, SubArray{T, N, A, I, true} where {T, N, A<:DenseArray, I<:Union{Tuple{Vararg{Real, N} where N}, Tuple{AbstractUnitRange, Vararg{Any, N} where N}}}, DenseArray}, MI<:Tuple{Vararg{Base.MultiplicativeInverses.SignedMultiplicativeInverse{Int64}, N} where N}}, DenseArray},SparseArrays.SparseMatrixCSC{Float64, Int64},Vector{Float64},Bool,Bool})   # time: 0.004844038
+Base.precompile(Tuple{typeof(setindex!),SparseArrays.SparseMatrixCSC{Float64, Int64},Any,Int64,Int64})   # time: 0.004176986
+Base.precompile(Tuple{typeof(setindex!),SparseArrays.SparseMatrixCSC{Float64, Int64},Float64,Int64,Int64})   # time: 0.003730561
+Base.precompile(Tuple{typeof(fill!),SparseArrays.SparseMatrixCSC{Float64, Int64},Int64})   # time: 0.003499388
+
+#Iterators
+Base.precompile(Tuple{typeof(iterate),Iterators.Zip{Tuple{Int64, Int64}}})   # time: 0.001524352
+
+#Jump Containers
+#Base.precompile(Tuple{Type{JuMP.Containers.DenseAxisArray},Core.Array{T, N},Any,Tuple{Vararg{_AxisLookup, N}} where {T,N}})
+
+#SparseArrays.HigherOrderFns
+Base.precompile(Tuple{typeof(SparseArrays.HigherOrderFns._sparsifystructured),Vector{Float64}})   # time: 0.004664484
+Base.precompile(Tuple{getfield(SparseArrays.HigherOrderFns, Symbol("#3#4")),Float64,Float64})   # time: 0.002634265
+isdefined(SparseArrays.HigherOrderFns, Symbol("#3#4")) && Base.precompile(Tuple{getfield(SparseArrays.HigherOrderFns, Symbol("#3#4")),BigFloat,Float64})   # time: 0.00150329
+isdefined(SparseArrays.HigherOrderFns, Symbol("#3#4")) && Base.precompile(Tuple{getfield(SparseArrays.HigherOrderFns, Symbol("#3#4")),Float64,Float64})   # time: 0.001362288
+
+#SentinelArrays
+Base.precompile(Tuple{typeof(Base.Broadcast.broadcasted),typeof(big),SentinelArrays.ChainedVector})   # time: 0.107109085
```

Comment:
@hannahtro where was this called? This is interesting because we didn't have SentinelArrays in our dependency